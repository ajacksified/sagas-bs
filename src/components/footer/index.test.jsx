import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter as Router, withRouter } from 'react-router';

import Footer from './index';

test('Outputs the expected tree', () => {
  const Component = withRouter(Footer);
  const component = renderer.create(<Router><Component /></Router>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
