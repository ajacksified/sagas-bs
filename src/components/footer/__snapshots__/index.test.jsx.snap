// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Outputs the expected tree 1`] = `
<footer
  className="footer inverted"
>
  <div
    className="container"
  >
    Copyright whenever, whoever.
    <a
      href="/privacy-policy"
      onClick={[Function]}
    >
      Privacy Policy
    </a>
    <a
      href="/terms-of-service"
      onClick={[Function]}
    >
      Terms of Service
    </a>
  </div>
</footer>
`;
